### PR TITLE
update managed instance SLA

### PR DESCRIPTION
**Pull request by bcreager**
Update managed instance SLA

### CODE REVIEW

**Review by kevinwojo — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/delivery/managed/index.md [hunk 1]
```diff
@@ -23,8 +24,18 @@ Please first read [the customer-facing managed instance documentation](https://d
...
+2. Message the team in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX)
+
+## SLAs for managed instances
+
+Support SLAs for Sev 1 and Sev 2 can be found [here](https://handbook.sourcegraph.com/departments/support/#slas). Other engineering SLAs are listed below
+
+|                                               | Description                                            | Response time                                  | Provide LOE                                                      | Resolution time                                                 |
+| --------------------------------------------- | ------------------------------------------------------ | ---------------------------------------------- | ---------------------------------------------------------------- | --------------------------------------------------------------- |
+| New instance Creation                         | Spin up new instance for a new customer                | Within 24 hours of becomeing aware of the need | Within 2 weeks to provide LOE (leve of effort)                   | Within 10 working days from agreement                           |
```
Comment (caugustus-sourcegraph) [inline content/departments/product-engineering/engineering/cloud/delivery/managed/index.md]:
```suggestion
| New instance Creation                         | Spin up new instance for a new customer                | Within 24 hours of becoming aware of the need | Within 2 weeks to provide LOE (level of effort)                   | Within 10 working days from agreement                           |
```

File: content/departments/product-engineering/engineering/cloud/delivery/managed/index.md [hunk 2]
```diff
@@ -46,15 +57,14 @@ With that said, Docker Compose deployments can scale up to the largest GCP insta
...
 - **Admin access**: Both the customer and Sourcegraph personnel will have access to an application-level admin account.
 - **VM/SSH access**: Only Sourcegraph personnel will have access to the actual GCP VM, this is done securely through GCP IAP TCP proxy access only. Sourcegraph personnel can make changes or provide data from the VM upon request by the customer.
 - **Inbound network access**: The customer may choose between having the deployment be accessible via the public internet and protected by their SSO provider, or for additional security have the deployment restricted to an allowlist of IP addresses only (such as their corporate VPN, etc.)
-- **Outbound network access**: The Sourcegraph deployment will have unfettered egress TCP/ICMP access, and customers will need to allow the Sourcegraph deployment to contact their code host. This can be done by having their code-host be publicly accessible, or by allowing the static IP of the Sourcegraph deployment to access their code host.
+- **Outbound network access**: The Sourcegraph deployment will have unfettered egress TCP/ICMP access, and customers will need to allow the:w
```
Comment (caugustus-sourcegraph) [inline content/departments/product-engineering/engineering/cloud/delivery/managed/index.md]:
```suggestion
- **Outbound network access**: The Sourcegraph deployment will have unfettered egress TCP/ICMP access, and customers will need to allow the
```

**Review by caugustus-sourcegraph — APPROVED** [review]

**Review by dan-mckean — APPROVED** [review]
I'm happy with it since Kevin and Crystal are.

File: content/departments/product-engineering/engineering/cloud/delivery/managed/index.md [hunk 3]
```diff
@@ -23,8 +24,18 @@ Please first read [the customer-facing managed instance documentation](https://d
 After [determining a managed instance is what a customer/prospect wants](https://docs.sourcegraph.com/admin/install/managed), Customer Engineers should:
 
 1. Submit a request to the Delivery team via the [Managed Instance Request](https://github.com/sourcegraph/customer/issues/new?assignees=&labels=team%2Fdelivery&template=new_managed_instance.md&title=) issue template in the sourcegraph/customer repo
-2. Add the issue to the "[Delivery](https://github.com/orgs/sourcegraph/projects/205/views/1)" board in GitHub (leave the status blank)
-3. Message the team in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX)
+2. Message the team in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX)
+
+## SLAs for managed instances
+
+Support SLAs for Sev 1 and Sev 2 can be found [here](https://handbook.sourcegraph.com/departments/support/#slas). Other engineering SLAs are listed below
```
Comment (dan-mckean) [inline content/departments/product-engineering/engineering/cloud/delivery/managed/index.md]:
```suggestion
Support SLAs for Sev 1 and Sev 2 can be found [here](../../../../../support/index.md#slas). Other engineering SLAs are listed below
```

**Pull request merged**